config/routes.rb

```diff
@@ -1,3 +1,6 @@
 Rails.application.routes.draw do
+  get 'startup/index'
```

Comment:
Try to do it with `resources` command too

Reply:
> Try to do it with `resources` command too

how do i do that?

Reply:
like line:4

Reply:
> like line:4

 `resources :trucks_controller, only: [:index]`
like this? or am i doing it wrongly